sstmac/hardware/network/network_message.cc

```diff
@@ -307,8 +318,9 @@ NetworkMessage::serialize_order(serializer& ser)
   ser.primitive(remote_buffer_);
   ser.primitive(local_buffer_);
   ser.primitive(smsg_buffer_);
-  ser & sstmac::array(wire_buffer_, payload_bytes_);
-  //this has to go here, weirdness with sstmac::array
+  if (!wire_is_null_) {
+    ser & sstmac::array(wire_buffer_, payload_bytes_);
+    }
```

Comment:
This appears to call `ser & sstmac::array(wire_buffer_, payload_bytes_);` twice if `wire_is_null != false`

Reply:
Resolved in CS 17bdc50